.github/workflows/linux.yml

```diff
@@ -94,12 +94,8 @@ jobs:
             version: 22.04
           - os: ubuntu
             version: 24.04
-          - os: debian
-            version: 11
           - os: debian
             version: 12
-          - os: fedora
-            version: 35

```

Comment:
Please also remove the readme links

Reply:
Ah, right!
There's also a few changes remaining I should implement here that were pointed to me by the people at the unicode mailing list anyway.

Reply:
This should be fixed now.